api/src/test/java/org/pmiops/workbench/workspaceadmin/WorkspaceAdminServiceTest.java

```diff
@@ -425,95 +399,17 @@ public void testListFilesAllFilesInBucket() {
   }
 
   @Test
-  public void testDeleteRuntimesInProject() {
-    List<LeonardoListRuntimeResponse> listRuntimeResponseList =
-        ImmutableList.of(testLeoListRuntimeResponse);
-    when(mockLeonardoNotebooksClient.listRuntimesByProjectAsService(GOOGLE_PROJECT_ID))
-        .thenReturn(listRuntimeResponseList);
-
-    workspaceAdminService.deleteRuntimes(
-        WORKSPACE_NAMESPACE,
-        new ListRuntimeDeleteRequest()
-            .runtimesToDelete(ImmutableList.of(testLeoRuntime.getRuntimeName())));
-    verify(mockLeonardoNotebooksClient)
-        .deleteRuntimeAsService(GOOGLE_PROJECT_ID, testLeoRuntime.getRuntimeName());
-    verify(mockLeonardoRuntimeAuditor)
-        .fireDeleteRuntimesInProject(
-            GOOGLE_PROJECT_ID,
-            listRuntimeResponseList.stream()
-                .map(LeonardoListRuntimeResponse::getRuntimeName)
-                .collect(Collectors.toList()));
-  }
-
-  @Test
```

Comment:
these 4 cases are no longer valid:
* DeleteSome - can only delete one
* DeleteDoesNotAffectOtherProjects - why would it?  we explicitly specify the project
* NoRuntimes - must specify one runtime
* NullRuntimesList - must specify one runtime